syntax = "proto3";

package greet;


service Blog {
	// Unary
	rpc SendNotification (CreateNotificationRequest) returns (NotificationReply);


}


message CreateNotificationRequest {
	string owner_id = 1;
	int32 type_id = 2;
	string content_id = 3;
	string content = 4;
}


message NotificationReply {
   string id = 1;
}
